e2e/profiles/profiles-both-profiles-with-settings.e2e.spec.ts

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright © 2023 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { test } from '@playwright/test';
+
+import {
+  clearUserProfile,
+  testForNoFirstRunWindow,
+  verifySettings,
+  verifySystemProfile,
+  verifyUserProfile,
+} from '../utils/ProfileUtils';
+
+test.describe.serial('sys-profile with settings', () => {
```

Comment:
We need to document the need for human involvement in these tests, as well as the details of what that involvement looks like. Here are some options I can think of:
1. Create a `e2e/profiles/README.md` with these details. Add a pointer to it in the root README.md.
2. Combine the profiles tests into one file, and document the details inside the root README.md. This option is my preference.